docs/platform_concepts/studio/train/entities.md

```diff
@@ -3,159 +3,188 @@ title : Entities
-:::
+2. Enter the entity name in **Entity name** and select **Type** as **Regex**.
 
-![](https://i.imgur.com/UEQWA4v.png)
+![](https://i.imgur.com/ujjr6yM.png)
 
----
+3. Enter the format in the **Regex** field.
 
-### 4.2 Regex
+## Train the bot on entities
 
-Cases when the pattern of the information is known but there is no way to list all the possible exact values of the same entity. This includes entities like Flight Number, ID cards, etc.
+Once you add entities, you need to train your bot on the same. To train entities:
 
-> **Policy Numbers** - OG - XXXX -XXX -XXXX
-> **PAN Number** - ABCDE1234F
+1. Click **Train entities** on top.
 
-Defining these entities is very simple, by entering the regular expression (general format of input) Example, Regex for PAN card- "[A-Z]{5}[0-9]{4}[A-Z]{1}".
+![](https://i.imgur.com/HVuCiMl.png)
 
----
-### 4.3 Text
 
-Text Type entities can be used when the use-case doesn't require list/regex type (when there is no specific list/format of entity items). 
+2. Select the **Model type**, the bot can be trained on English and multiple languages.
 
-* Add **Entity name** and **Type**.
+  > Multilanguage training  works only for **Text** type entities.
 
-![](https://i.imgur.com/m8WfLmL.png)
+![](https://i.imgur.com/RnA8fYY.png)
 
-* Open **Studio** > **Train** > **Intents** and [add intents](https://docs.yellow.ai/docs/platform_concepts/studio/train/intents#add-intent).  
-* In each Intent- user utterance, select a word that hold the meaning of an entity and right click. Select the required entity.
+3. Enable **Fuzzy search** for the bot to conduct searches for text that closely matches a term, even when there are slight misspellings. 
 
-![](https://i.imgur.com/yg1p3Z6.jpg)
+For instance, if you perform a fuzzy search for "rode," it will identify terms with similar spelling, such as "ride" or "node," rather than strictly requiring an exact match.
 
+6. Click **Train**.
 
-:::note
-It is highly recommended not to use text entity unless required as it is not very definite. 
-:::
 
----
+## Update entities
 
-### 4.4 System entities
+You can update the entities(all of the information in it) at any point of time. To update an entity:
 
-Apart from the List and Pattern type entities, a set of entities are recognized by the model out-of-the-box without any specific training. These include:
+1. Go to **Studio** > **Train** > **Entities**.
 
-  - Name
-  - Date
-  - Email
-  - Location
+![](https://i.imgur.com/bJNtr5w.png)
 
+2. Click **Edit**.
+3. Make the required changes and click **Update**.
 
+![](https://i.imgur.com/gktslXm.png)
 
 
+## Import entities
 
+You can import the entities from an external source. The file to be uploaded should be a CSV file with two headers, **Name** for the item name and **Synonyms** for comma-separated synonyms associated with the item.
```

Comment:
Why is the entity values column called synonyms? 
just curious...

Reply:
Had the same doubt, will talk to the product team